package/doc/sphinx/source/documentation_pages/coordinates/pickle_universe.rst

```diff
@@ -0,0 +1,117 @@
+.. Contains the formatted docstrings for the serialization of universe located 
+.. mainly in 'MDAnalysis/libs/pickle_file_io.py'
```

Comment:
I agree with @kain88-de that this document is mixing user and developer docs.

My suggestion is to focus _here_ on the _developers_ only and in particular on the serialization of coordinate readers. You can say a little bit about serializing Universe (as this is the background and because we don't really have a better place in the docs yet) but then focus on the technical aspects of 
- what happens when a reader is serialized
- what needs to be done for a new reader to make serialization work

Reply:
Take out the "how to do things with multiprocessing" – that's a user part. Instead of adding it somewhere in the MDAnalysis API docs, create a new chapter in the User Guide instead – @lilyminium should be able to help and there's [Contributing to the User Guide](https://userguide.mdanalysis.org/1.0.0/contributing_docs.html).

Along the lines of #1175 we should start making an effort to split the docs into API docs ("the docs" at https://docs.mdanalysis.org) and the user docs ("User Guide" https://userguide.mdanalysis.org).

Reply:
Hmm, it could go either in the "analysis" or the "advanced" sections? Example of a tutorial in the "advanced" section: [Adding a Residue or Segment to a Universe](https://userguide.mdanalysis.org/1.0.0/advanced_topology.html). And in "analysis": [Writing your own trajectory analysis](https://userguide.mdanalysis.org/1.0.0/examples/analysis/custom_trajectory_analysis.html)

> we should start making an effort to split the docs into API docs

Yes, I think this should probably come after or with separating the user guide into "user" and "dev" guides, though...

Reply:
>> we should start making an effort to split the docs into API docs

I primarily meant that we can already create "How to do X" (such as "How to run do parallel processing") in the User Guide as opposed to also first have them in the docs (which I referred to as "API docs") and then move them later.

Reply:
I would put the user-portion of your parallel analysis docs in the **analysis** section of the User Guide – you can start a section _Parallel Analysis_ with your general introduction and then your example how to use multiprocessing.  You can add a See Also section for PMDA for the time being. Later, sections on dask, MPI, and whatnot can be added.